addons/mail/models/mail_activity.py

```diff
@@ -21,7 +21,7 @@ class MailActivity(models.Model):
     field that indicates the activity linked to the message. """
     _name = 'mail.activity'
     _description = 'Activity'
-    _order = 'date_deadline ASC'
+    _order = 'date_deadline, id'
```

Comment:
"date_deadline ASC, id ASC" ?

Reply:
Isn't ASC the order by default ?

Reply:
Would say better explicit than implicit. That's the kind of thing a lot of people always wonder which is the default value and have to go check on internet... does not cost much to be explicit ^^ .

Reply:
I don't agree but like you said, it cost nothing and I don't really care anyway so I made the change :+1: